Pipeline/PreprocessCombinedStudies.R

```diff
@@ -0,0 +1,150 @@
+#title: "MEP-LINCs Preprocessing"
+#author: "Mark Dane"
+# 2017
+## Example Command line call Rscript PreprocessCombinedStudies.R --verbose --inputPath syn7494072 --synapseStore syn5713302 HMEC122L_SS1 HMEC 122L_SS4
+library(MEMA)#merge, annotate and normalize functions
+library(parallel)#use multiple cores for faster processing
+library(RUVnormalize)
+library(ruv)
+library(stringr)
+library(tidyr)
+library(readr)
+library(dplyr)
+suppressPackageStartupMessages(library(optparse))
+suppressPackageStartupMessages(library(synapseClient))
+
+getCSCommandLineArgs <- function(){
+  option_list <- list(
+    make_option(c("-v", "--verbose"), action="store_true", default=FALSE,
+                help="Print extra output"),
+    make_option(c("-k", "--k"), type="integer", default=256,
+                help="Number of factors to use in RUV normalization [default %default]",
+                metavar="number"),
+    make_option(c("-i", "--inputPath"), type="character", default=NULL, metavar="PATH",
+                help="Path to local input data directory or Synapse ID for a File View."),
```

Comment:
make_option(c("-i", "--inputSynID"), type="character", default=NULL, metavar="synID",
                help="Synapse ID for a File View with input data."),